openapi-cli/src/test/resources/generators/client/swagger/header_integer_signed32.yaml

```diff
@@ -0,0 +1,102 @@
+openapi: "3.0.0"
+info:
+  version: 1.0.0
+  title: OpenApi Petstore
+  license:
+    name: MIT
+servers:
+  - url: http://petstore.{host}.io/v1
+    description: The production API server
+    variables:
+      host:
+        default: openapi
+        description: this value is assigned by the service provider
+
+tags:
+  - name: pets
+    description: Pets Tag
+  - name: list
+    description: List Tag
+
+paths:
+  /pets:
+    get:
+      summary: Info for a specific pet
+      operationId: showPetById
+      tags:
+        - pets
+      parameters:
+        - in: header
+          description: Tests header 01
+          name: X-Request-ID
+          schema:
+            type: integer
+            format: int32
+          required: true
+        - in: header
+          description: Tests header 02
+          name: X-Request-Client
+          schema:
+            type: array
+            items:
+              type: integer
+              format: int32
+          required: true
+        - in: header
+          description: Tests header 03
+          name: X-Request-Pet
+          schema:
+            type: array
+            items:
+              $ref: "#/components/schemas/Pet"
+          required: true
+        - in: header
+          description: Tests header 04
+          name: X-Request-Header
+          schema:
+            type: integer
+            format: invalid_format
```

Comment:
Here when invalid format is given, can we print any warning in the outstream ? What is the current behaviour ?